File: php/04-control-flow/conditionals.php
<?php
$score = 82;

if ($score >= 90) {
    echo "Grade: A\n";
} elseif ($score >= 80) {
    echo "Grade: B\n";
} elseif ($score >= 70) {
    echo "Grade: C\n";
} else {
    echo "Grade: F\n";
}

// Truthiness: weak typing coerces values to booleans
$input = "0";
if ($input) {
    echo "Truthy\n";
} else {
    echo "Falsy: the string \"0\" is considered false\n";
}

// Logical operators combine conditions
$age = 25;
$hasTicket = true;
if ($age >= 18 && $hasTicket) {
    echo "Entry granted\n";
}

File: python/04-control-flow/conditionals.py
# Branching with if / elif / else
temperature = 72

if temperature > 85:
    print("It's hot outside")
elif temperature >= 60:
    print("The weather is pleasant")
else:
    print("Bring a jacket")

# Python evaluates "truthiness" — empty values are falsy
items = []
if items:
    print(f"You have {len(items)} items")
else:
    print("Your cart is empty")

# Comparison and logical operators combine cleanly
age = 25
has_ticket = True
if age >= 18 and has_ticket:
    print("Entry granted")

# Membership tests read like English
fruit = "apple"
if fruit in ("apple", "banana", "cherry"):
    print(f"{fruit} is in stock")
